tutorials/compute-quickstart.md

```diff
@@ -0,0 +1,280 @@
+```
+
+Install the app dependencies:
+
+```bash
+cd todomvc-mongodb; npm install
+```
+
+```bash
+sed -i -e 's/8080/80/g' server.js
+```
+
+Start the to-do web app:
+
+```bash
+sudo nohup nodejs server.js --be_ip [backend-internal-ip] --fe_ip [frontend-internal-ip] &
+```
+
+Replace `[frontend-internal-ip]` and `[frontend-internal-ip]` with 
+[internal IP addresses][spotlight-internal-ip] for the instances that you created.
+These IP addresses are listed for each instance in the **VM instances** table.
+
+Exit the SSH session and disconnect from the frontend instance:
+
+```bash
+exit
+```
+
+### Visit the web app
+
+Visit your webserver at the [external IP address][spotlight-external-ip] listed next to your frontend
+instance in the **VM instances** table.
+
+## Cleanup
+
+To remove your instances, select the [checkbox][spotlight-instance-checkbox]
+next to your instance names and click the [**Delete**][spotlight-delete-button] button.
+
+## Conclusion
+
+<walkthrough-conclusion-trophy></walkthrough-conclusion-trophy>
+
+You're done!
+
+Here's what you can do next:
+
+*   Find Google Cloud Platform
+    [samples on GitHub](http://googlecloudplatform.github.io/).
+
+*   Learn how to set up
+    [Cloud Load Balancing](https://cloud.google.com/compute/docs/load-balancing/).
+
+*   Learn how to
+    [transfer files to your virtual machine](https://cloud.google.com/compute/docs/instances/transfer-files/).
+
+*   Learn how to
+    [run containers](https://cloud.google.com/compute/docs/containers).
+
+[pricing]: https://cloud.google.com/compute/#compute-engine-pricing
+[spotlight-instance-name]: walkthrough://spotlight-pointer?spotlightId=gce-vm-add-name
+[spotlight-instance-zone]: walkthrough://spotlight-pointer?spotlightId=gce-vm-add-zone-select
+[spotlight-create-instance]: walkthrough://spotlight-pointer?spotlightId=gce-zero-new-vm,gce-vm-list-new
+[spotlight-boot-disk]: walkthrough://spotlight-pointer?cssSelector=vm-set-boot-disk
+[spotlight-firewall]: walkthrough://spotlight-pointer?spotlightId=gce-vm-add-firewall
+[spotlight-vm-list]: walkthrough://spotlight-pointer?cssSelector=vm2-instance-list%20.p6n-checkboxed-table
+[spotlight-control-panel]: walkthrough://spotlight-pointer?cssSelector=#p6n-action-bar-container-main
+[spotlight-ssh-buttons]: walkthrough://spotlight-pointer?cssSelector=gce-connect-to-instance
+[spotlight-notification-menu]: walkthrough://spotlight-pointer?cssSelector=.p6n-notification-dropdown,.cfc-icon-notifications
+[spotlight-console-menu]: walkthrough://spotlight-pointer?spotlightId=console-nav-menu
+[spotlight-open-devshell]: walkthrough://spotlight-pointer?spotlightId=devshell-activate-button
+[spotlight-machine-type]: walkthrough://spotlight-pointer?spotlightId=gce-add-machine-type-select
+[spotlight-submit-create]: walkthrough://spotlight-pointer?spotlightId=gce-submit
+[spotlight-internal-ip]: walkthrough://spotlight-pointer?cssSelector=gce-internal-ip
```

Comment:
This internal IP address spotlight item doesn't seem to be working.

Reply:
It would be fine to trim it out for the time being. I can look into whether there is a new selector that would work, or perhaps a new ID would need to be added to the UI.

Reply:
I'll trim it out. Thanks.